ui/js/page/wallet.js

```diff
@@ -16,20 +16,20 @@ var AddressSection = React.createClass({
       this.setState({
         address: address,
       });
-    });
+    }.bind(this));
```

Comment:
Because it doesn't use an arrow function and thus loses the scope of `this` otherwise. Wherever you see `.bind(this)` it can probably be re-written as an arrow function.